services/infra-service/infra_service.py

```diff
@@ -8,12 +8,12 @@
 
 logger = logging.Logger(__name__)
 
-DB_URL = os.getenv('DB_URL', 'http://127.0.0.1:3306/') 
+DB_URL = os.getenv('DB_URL', 'resilientsix-mysql-db.mysql.database.azure.com')
```

Comment:
these need to stay pointed towards localhost (default value). The aks value will be passed by ENV via AKV

Reply:
fixed